### PR TITLE
Place matching docs

### PR DESCRIPTION
**Pull request by jsani-radar**
Docs for Place matching launch

TODO:
- [x] insert images for each geofence type transformation

place_matching

### CODE REVIEW

File: docs/dashboard.mdx [hunk 1]
```diff
@@ -47,3 +47,28 @@ The User Activity page is the most straightforward way to debug a user's recent
 - **`VISIT_DEPARTURE`** (iOS only): location update triggered by a departure [iOS visit monitoring service](https://developer.apple.com/documentation/corelocation/getting_the_user_s_location/using_the_visits_location_service)
 - **`MANUAL_LOCATION`**: location update triggered by `Radar.trackOnce` passing in a location not collected by the Radar SDK
 - **`MOCK_LOCATION`**: location update triggered by `Radar.mockTracking`
+
+## Place matching
+
+Place matching is the simplest way to clean your geofence datasets while making sure they stay up-to-date and accurate over time. Navigate to the [Geofences](https://radar.com/dashboard/geofences) page and select *Import*. If [Places](/places) are enabled for your project, opt-in to this feature by toggling on *Enable place matching*. Select [chains](/places/chains) to match your Geofence dataset against. The import process will attempt to match each geofence against Radar's places for those chains.
```
Comment (bschoeneweis) [inline docs/dashboard.mdx:53]: nit: seems like `Geofences` is capitalized as a context type but `places` generally are not.  May want to make these consistent.  Can ignore if this is the precedent across the docs site already

**Review by bschoeneweis — CHANGES_REQUESTED** [review]
Looks great!  Few small nits, love the gif though.  This will definitely help guide usage 🙌

File: docs/dashboard.mdx [hunk 2]
```diff
@@ -47,3 +47,28 @@ The User Activity page is the most straightforward way to debug a user's recent
...
+
+### How it works
+
+1. **Import your Geofences against Radar's places data.** Upon importing your geofences into Radar, select [chains](/places/chains) to match your geofences aganist. By default, Radar will search for [places](/places) for those chains within 10 kilometers (~6 miles) of each Geofence. The search radius can be configured as an advanced option at the time of import.
```
Comment (corypisano) [inline docs/dashboard.mdx:57]: do we want to mentioned re: search radius that it will match to the closest one within 10 km

**Review by corypisano — APPROVED** [review]
lgtm after bradley's comments

**Review by bschoeneweis — APPROVED** [review]
Looks great!

**Pull request merged**